src/main/java/ChatBot.java

```diff
@@ -0,0 +1,53 @@
+import esme.Ui;
+
+import java.util.Scanner;
+
+public class ChatBot {
+    private Scanner in;
+    private Ui ui;
+
+    public ChatBot() {
+        this.in = new Scanner(System.in);
+        this.ui = new Ui();
+    }
+
+    public void start() {
+        ui.greet();
+        String line;
+
+        while (true) {
+            line = in.nextLine();
+            String[] words = line.split(" ");
+            if (words.length == 0) {
+                ui.promptEmptyInput();
+            }
```

Comment:
Maybe can add a ``continue`` statement here to avoid ``words[0]`` being accessed by subsequent ``word[0]``?